resources/sushy-tools/redfish-emulator.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eux -o pipefail
+
+CONFIG="${SUSHY_TOOLS_CONFIG:-/root/sushy/conf.py}"
+ARGS=
+
+if [[ -f "${CONFIG}" ]]; then
```

Comment:
```suggestion
if [[ -r "${CONFIG}" ]]; then
```
Maybe `-r` so the file exists and is readable too? Doesn't matter right now as its running as`root` now, so feel free to ignore too.

Reply:
I did this on purpose, I want the service to fail if the rights are not set up correctly (it can happen even with root if selinux is enforcing).

Reply:
If we know it is a real possibility to have file existing but not readable, why not explicitly fail and exit here already? Did not check what sushy-tools does in such case, but in general, services may exit gracefully, dump stack or even just ignore the failing config file.